volgen/templates/gfproxy.graph

```diff
@@ -0,0 +1,10 @@
+protocol/server
+debug/io-stats
```

Comment:
To name io-stats as "gfproxy-*" or with volname, instead of the normal "<volname-io>-stats", add the name you want here. For eg. this line could be,
`debug/io-stats, gfpoxy-{{ volname }}`

Reply:
Done